### DISCUSSION TITLE
Where can one download ImageMagick-6.5.6-8-Q8-windows-dll.exe

### DISCUSSION THREAD

**Discussion by ippezshelby** [ImageMagick/ImageMagick#5459] [Help]
Where can one download version of ImageMagick-6.5.6-8-Q8-windows-dll.exe?. It used to be bundled at http://rubyforge.org/frs/download.php/64917/RMagick-2.12.0-ImageMagick-6.5.6-8-Q8.zip, it seems server is down.

I need the above version for some project
![image](https://user-images.githubusercontent.com/13125234/185519910-2761eb72-c1e7-4f06-8057-c6d4581851c6.png)

**Comment by dlemstra** [comment]
We are not the maintainers of the RMagick project so we cannot help you with this. But I would recommend you to upgrade to a more recent version of RMagick and ImageMagick. You could probably download the latest version of ImageMagick and of RMagick to accomplish the same result?

**Reply by ippezshelby** [reply]
@dlemstra thanks for your reply. I have downloaded and installed latest version of [ImageMagick-7.1.0-46-Q16-x64-dll.exe](https://imagemagick.org/archive/binaries/ImageMagick-7.1.0-46-Q16-x64-dll.exe) but when i try to upload an image (.png, .bmp, .jpg) from the application (fedena) it throws error as below.....

![image](https://user-images.githubusercontent.com/13125234/185564744-f2e7fef1-b263-4cee-bfb3-3205179cb4da.png)

From fedena [installation manual](https://en.wikibooks.org/wiki/Fedena/Installation), they refer to a particular version **ImageMagick-6.5.6-8-Q8-windows-dll.exe** otherwise RMagick  is only a gem file which i have already installed in ruby 1.8.7

If only i can find download link for specified version of **ImageMagick-6.5.6-8-Q8-windows-dll.exe**, it will help solve my problem

**Reply by dlemstra** [reply]
That version is ancient and should no longer be used. We also cannot help you with getting ImageMagick to work with that application. The only tip that I could give is to download the latest ImageMagick 6 version instead. The version of RMagick that you are using is probably incompatible with ImageMagick 7.
